src/main/java/io/candydoc/ddd/annotations/DomainEntity.java

```diff
@@ -0,0 +1,13 @@
+package io.candydoc.ddd.annotations;
+
+import java.lang.annotation.*;
+
+@Documented
+@Inherited
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface DomainEntity {
+  String name() default "";
```

Comment:
Do you think it's relevant to have a custom name ? (not saying I would remove it, but I'm curious to know your opinion)

Reply:
At the beginning, we get inspired by the template of the aggregate. However, now, it seems that keeping the name isn't relevant for an entity. We're gonna remove it